FinSearchUnified/Bundle/SearchBundleFindologic/FacetHandler/TextFacetHandler.php

```diff
@@ -122,11 +122,9 @@ private function getValueListItems(FacetInterface $facet, Criteria $criteria, Si
      */
     private function createValueListFacetResult(FacetInterface $facet, Criteria $criteria, SimpleXMLElement $filter)
     {
-        if (!$filter->items->item) {
-            return null;
-        }
+        $items = $filter->items->item;
```

Comment:
Why was this variable introduced? It is not consistent with `createRadioFacetResult`.

Reply:
@TheKeymaster you told me in a previous review to introduce this variable which checks for the null value